standard/info/wikis/omegastrikers/info.lua

```diff
@@ -29,4 +29,5 @@ return {
 
 	defaultTeamLogo = 'Omega Strikers default allmode.png', ---@deprecated
 	defaultTeamLogoDark = 'Omega Strikers default allmode.png', ---@deprecated
+	match2 = 0,
```

Comment:
```suggestion
	match2 = 2,
```
although not on git ...
(new wiki with only match2)

Reply:
If it's not on git, it doesn't have match2 (officially)

Reply:
If it's not on git, it doesn't have match2 (officially)